app/assets/stylesheets/profile.scss

```diff
@@ -1,29 +1,28 @@
 #profile_container {
   .profile_header {
     margin-bottom: 15px;
-    background-color: $framed-background;
-    padding-left: 10px;
-    padding-right: 10px;
-    padding-top: 20px;
-    margin-top: -20px;
-    box-shadow: $card-shadow;
+    padding: 5px;
+    margin-top: 10px;
+    text-shadow: 1px 1px 0px #fff;
 
     #edit_profile, #unblock_user_button, .aspect_dropdown {
       margin-top: 15px;
+      text-shadow: none;
     }
 
     #author_info {
       h2 {
         line-height: 35px;
-        margin-top: 10px;
+        margin-top: 0px;
```

Comment:
- Properties should be ordered line-height, margin-bottom, margin-top
- `0px` should be written without units as `0`